lib/healthchecks.js

```diff
@@ -121,6 +121,8 @@ function get(url, headers, resolver, timeout) {
     })
   };
 
+  state = state || { redirects: 0 };
```

Comment:
It was a symptom of the ugly solution I implemented at the beginning using recursion. Upgraded to an async loop :sunglasses: